frontend/components/RegistrationPage.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ChangeEvent } from 'react';
 
 type RegistrationPageProps = {
- formData: { email: string; password: string; firstName: string; lastName: string; bio: string; country: string };
+ formData: { email: string; password: string; confirmPassword: string; firstName: string; lastName: string; bio: string; country: string; personalLink: string; profilePictureLink: string; };
```

Comment:
Sending the confirmPassword is not necessary. We only use it to check in the frontend if the password and the confirmPassword match. It doesn't get sent to the backend. If there's still confirmPassword in the backend, we need to remove it.